pkgs/by-name/bi/bilibili/package.nix

```diff
@@ -1,29 +1,40 @@
-{ lib
-, stdenv
-, fetchurl
-, electron
-, zstd
-, makeWrapper
+{
+  lib,
+  stdenv,
+  fetchurl,
+  electron_30,
+  dpkg,
+  makeWrapper,
+  commandLineArgs ? "",
```

Comment:
Still haven't seen a proof of this being necessary.
```suggestion
```

Reply:
@Bot-wxt1221 is there a use case why you want to add `commandLineArgs` here ?